src/api/spec/factories/project.rb

```diff
@@ -3,6 +3,14 @@
     sequence(:name) { |n| "project_#{n}" }
     title { Faker::Book.title }
 
+    after(:create) do |project|
+      # NOTE: Enable global write through when writing new VCR cassetes.
+      # ensure the backend knows the project
+      if CONFIG['global_write_through']
+        Suse::Backend.put("/source/#{CGI.escape(project.name)}/_meta", project.to_axml)
```

Comment:
@hennevogel This actually fails when global_write_through got enabled.

```
     NoMethodError:
       undefined method `login' for nil:NilClass
     # ./app/models/project.rb:830:in `write_to_backend'
     # ./app/models/project.rb:879:in `block in store'
     # ./app/models/project.rb:877:in `store'
     # ./spec/factories/project.rb:10:in `block (3 levels) in <top (required)>'
```

And I wouldn't want to monkey patch this, just make it pass.